qfieldsync/core/cloud_transferrer.py

```diff
@@ -618,12 +693,19 @@ def __init__(
         self.transfer_type = transfer_type
 
         for file in self.files:
+            if use_file_local_dir:
+                destination = Path(file.local_dir).joinpath(file.name)
+            elif self.temp_dir:
```

Comment:
I guess there should be an assert in the `else` part in case the `if` and `elif` are not met?

Reply:
That's an artifact from previous logic, it should just be else: here, changed.